src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -48,10 +50,22 @@
     private TreeTableColumn<BoxObject, String> sizeColumn;
     @FXML
     private TreeTableColumn<BoxObject, String> dateColumn;
+    @Inject
+    ClientConfiguration clientConfiguration;
 
     @FXML
+
     public void initialize(URL location, ResourceBundle resources) {
 
+        if (clientConfiguration.hasAccount()) {
+            PRIVATE_KEY = clientConfiguration.getSelectedIdentity().getPrimaryKeyPair().getPrivateKey().toString();
+            PRIVATE_KEY = "77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a";
+        } else {
+            PRIVATE_KEY = "77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a";
+        }
+
+        System.out.print(PRIVATE_KEY);
```

Comment:
?
* hasAccount() is not related to getSelectedIdentity()
* the `if` has no effect ... result is always `PRIVATE_KEY = "77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a";`
* Debug output...